### ISSUE TITLE
CP/M through-hole cartridges stability

### ISSUE THREAD

**Issue by schenkmi** [Jaystonian/Cartridges#1]
Hi,
i've rebuild a CP/M through-hole cartridges, but it looks that it's not running stable. It only startup on my C64-C and shorty after I get weird characters on the screen. Any idea what I'm doing wrong ?
Cheers
Michael

**Comment by Jaystonian** [comment]
Which version did you build, the shorty cp/m?On 17 Aug. 2020 8:45 am, Michael Schenk <notifications@github.com> wrote:
Hi,
i've rebuild a CP/M through-hole cartridges, but it looks that it's not running stable. It only startup on my C64-C and shorty after I get weird characters on the screen. Any idea what I'm doing wrong ?
Cheers
Michael

**Comment by Jaystonian** [comment]
ok looks like you built the "true" version. Where did you find a 74LS12? CP/M carts only worked in silver-label c64 (very first edition) and all C128, which is faster than a 128 internal cpm mode.On 17 Aug. 2020 10:09 am, jaystonian@hotmail.com wrote:Which version did you build, the shorty cp/m?On 17 Aug. 2020 8:45 am, Michael Schenk <notifications@github.com> wrote:
Hi,
i've rebuild a CP/M through-hole cartridges, but it looks that it's not running stable. It only startup on my C64-C and shorty after I get weird characters on the screen. Any idea what I'm doing wrong ?
Cheers
Michael

**Comment by schenkmi** [comment]
Hi,
yes it's the true version. Regarding the 74LS12 I got them from Germany via eBay. So this card will only run on silver-label C64 which for sure I don't own. Do you have a clue why ? I have A C128 and will try it there. They shorty works with all C64 or still only with the silver-label ?
Cheers
Michael

**Comment by schenkmi** [comment]
It works on my C128. I've took a short video which can be viewed under https://youtu.be/MPaJR98bfZg.
On https://dfarq.homeip.net/commodore-64-cpm-operating-system/ I've found the information that it only works on silver-label C64.
Cheers
Michael

**Comment by Jaystonian** [comment]
The shorty is the exact same circuit except the ls12 is replaced by a 10 and a 3g06. Bil Herd suggested the cpm cart didnt work in 64s because of the power supply, but I've tried using HCT chips and other low power chips with a custom power supply, I can't get it working on a 64, maybe its the z80 itself drawing too much power.I like how it runs super fast on a c128 instead of its internal z80. Combine with jiffydos and its a very fast CP/M system.On 17 Aug. 2020 11:59 am, Michael Schenk <notifications@github.com> wrote:
It works on my C128. I've took a short video which can be viewed under https://youtu.be/MPaJR98bfZg.
On https://dfarq.homeip.net/commodore-64-cpm-operating-system/ I've found the information that it only works on silver-label C64.
Cheers
Michael
